pkg/pwalk/pwalk.go

```diff
@@ -29,16 +29,24 @@ type WalkFunc = filepath.WalkFunc
 // - if more than one walkFn instance will return an error, only one
 // of such errors will be propagated and returned by Walk, others
 // will be silently discarded.
+//
+// Deprecated: use github.com/opencontainers/selinux/pwalkdir.Walk
```

Comment:
Do you mean `github.com/opencontainers/selinux/pkg/pwalkdir.Walk`

Reply:
Ah, yes,  missed the `pkg/`